main.nf

```diff
@@ -487,7 +468,7 @@ process get_software_versions {
     """
 }
 
-if (params.compressedReference){
+if(hasExtension(params.fasta, 'gz')){
```

Comment:
I assumed since we only allow either all references to be compressed that this is enough to check for but as you mentioned there could be a `--fasta false` stuff happening... good catch! 👍